packages/vscode-extension/src/commands/CreateProject.ts

```diff
@@ -124,9 +125,8 @@ export default class CreateProject extends BaseAction {
 			commandArgs[COMMAND_ARGUMENTS.PROJECT_VERSION] = projectVersion;
 
 			commandArgs[COMMAND_ARGUMENTS.PROJECT_FOLDER_NAME] = publisherId + '.' + projectId;
```

Comment:
Should we also trim publisherId and projectId ?

Reply:
It looks like a good idea

Reply:
But it's not needed, since into the validator we already check the fields have no spaces with the validator: validateFieldHasNoSpaces